docs/pages/access-controls/access-request-plugins/access-monitoring-rules.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Routing Access Request notifications
+description: How to set up Teleport's Access Monitoring Rules to route Access Request notifications
+---
+
+With Teleport's Access Monitoring Rules Access Request notifications
+can be routed to plugins based on based on several conditions.
+For example, if you wish to notify different recipients e.g Slack channels,
+based on what roles or resources are being accessed, or based on what user
+is making the request.
+
+Currently only the Slack access plugin is supported.
+
+## Prerequisites
+
+- A managed Teleport Enterprise account.
```

Comment:
What is a managed account? Does this mean cloud?

Reply:
yes